libobs/audio-monitoring/osx/coreaudio-enum-devices.c

```diff
@@ -25,16 +25,15 @@ static bool obs_enum_audio_monitoring_device(obs_enum_audio_device_cb cb,
 
 	AudioObjectPropertyAddress addr = {
 		kAudioDevicePropertyStreams,
-		kAudioDevicePropertyScopeInput,
+		kAudioDevicePropertyScopeOutput,
 		kAudioObjectPropertyElementMaster
 	};
 
-	/* check to see if it's a mac input device */
-	if (!allow_inputs) {
-		AudioObjectGetPropertyDataSize(id, &addr, 0, NULL, &size);
-		if (size)
-			return true;
-	}
+	/* Check if the device is capable of audio output
```

Comment:
You should add punctuation to the end of each sentence.  You can probably omit the text "NOTE:" from this comment and just leave the proceeding comment text in place.